resources/release/rc-details-template.md

```diff
@@ -0,0 +1,130 @@
+## See OpenSearch RC ${OPENSEARCH_RC_NUMBER} and OpenSearch-Dashboards RC ${OPENSEARCH_DASHBOARDS_RC_NUMBER} details
+<details><summary>OpenSearch ${OPENSEARCH_RC_NUMBER} and OpenSearch-Dashboards ${OPENSEARCH_DASHBOARDS_RC_NUMBER} details</summary>
+<p>
+ ## OpenSearch ${OPENSEARCH_RC_BUILD_NUMBER} and OpenSearch-Dashboards ${OPENSEARCH_DASHBOARDS_RC_BUILD_NUMBER} is ready for your test.
+
+OpenSearch - [Build ${OPENSEARCH_RC_BUILD_NUMBER}](https://build.ci.opensearch.org/blue/organizations/jenkins/distribution-build-opensearch/detail/distribution-build-opensearch/${OPENSEARCH_RC_BUILD_NUMBER}/pipeline)
+OpenSearch Dashboards - [Build ${OPENSEARCH_DASHBOARDS_RC_BUILD_NUMBER}](https://build.ci.opensearch.org/blue/organizations/jenkins/distribution-build-opensearch-dashboards/detail/distribution-build-opensearch-dashboards/${OPENSEARCH_DASHBOARDS_RC_BUILD_NUMBER}/pipeline)
+
+- Use the following Docker-Compose to setup a cluster
+  <details><summary>docker-compose.yml</summary>
```

Comment:
Note that these will only support 2.x or above.
Since we do not have 1.x anymore, we should be fine with this docker compose file setup.

Thanks

Reply:
For future or more we can add multiple templates to frame the message as per the `major.x` in the future.